lib/fs.js

```diff
@@ -1452,7 +1457,13 @@ function lutimesSync(path, atime, mtime) {
 
 function writeAll(fd, isUserFd, buffer, offset, length, signal, callback) {
   if (signal?.aborted) {
-    callback(lazyDOMException('The operation was aborted', 'AbortError'));
+    if (isUserFd) {
+      callback(lazyDOMException('The operation was aborted', 'AbortError'));
+    } else {
+      fs.close(fd, function() {
+        callback(lazyDOMException('The operation was aborted', 'AbortError'));
+      });
+    }
```

Comment:
Calling `fs.close` with no callbacks defaults to throwing an exception, which is probably what we want here?
```suggestion
    if (!isUserFd) {
      fs.close(fd);
    }
    callback(lazyDOMException('The operation was aborted', 'AbortError'));
```

Reply:
I prefer it written in the other, because it's the same pattern that's used below in the write callback.